### PR TITLE
Wine: Update to version 6.0

### PR DESCRIPTION
**Pull request by EchterAgo**
This brings everything into one `RUN` command, saving disk space. This also removes the download of the signing keys and just uses `COPY` to copy them from the host. This also switches the base image to the same as the AppImage builder.

### CODE REVIEW

**Comment by EchterAgo** [comment]
https://ec.loping.net/4.2.3-106-gbe8f4ca8f/

**Comment by EchterAgo** [comment]
I tried on two machines which both reproduced the same wine build.

File: contrib/build-wine/docker/Dockerfile [hunk 1]
```diff
@@ -1,51 +1,54 @@
...
-        wget=1.19.4-1ubuntu2.2 \
         gnupg2=2.2.4-1ubuntu1.3 \
-        dirmngr=2.2.4-1ubuntu1.3 \
-        python3-software-properties=0.96.24.32.1 \
-        software-properties-common=0.96.24.32.1
-
-RUN apt-get update -q && \
-        apt-get install -qy \
+        software-properties-common=0.96.24.32.14 && \
+    echo "78b185fabdb323971d13bd329fefc8038e08559aa51c4996de18db0639a51df6 /tmp/winehq.key" | sha256sum -c - && \
```
Comment (cculianu) [inline contrib/build-wine/docker/Dockerfile:27]: Clever technique.

**Comment by cculianu** [comment]
Built on macOS docker host -- also all hashes match now (after I was sure to be on the same tag as you).  Great change man! Thanks so much! And also paving the way for newer python, etc.. Awesome work man!!

**Pull request merged**